### ISSUE TITLE
'failed: Invalid frame header' error

### ISSUE THREAD

**Issue by noontage** [faye/faye-websocket-ruby#112]
Hi
As the title says, occurred error  'failed: Invalid frame header'

I tried as follows

1. clone your latest master branch
2. move current directory to 'example'
3. install gems by bundlr ('bundle install')
4. access to 'localhost:9292' by chrome
5. open developper tool of chrome and  type `const ws = new WebSocket('ws://localhost:9292')`
     (console tab)
6. occur error `VM131:1 WebSocket connection to 'ws://localhost:9292/' failed: Invalid frame header`

tried environment:
  * OSX Sierra 
  * Google Chrome 65
  * ruby 2.3.1p112 / ruby 2.4.2
  * cleaned gem (removed all gem) before execute them.

**Comment by jcoglan** [comment]
I'm not able to reproduce this error. To run the example server, you should run `ruby example/server.rb` in the root of the project. Then visit `http://localhost:7000/ws.html`, and if you see messages appearing it means the server is working.

**Comment by jcoglan** [comment]
@noontage is this an ongoing problem, or can I close this issue as I was not able to reproduce your report?

**Comment by sdubinsky** [comment]
I got an 'invalid frame header' error because I was running rackup in dev mode.  Perhaps that was OP's problem?  Might be worth adding it to the readme in the relevant places.

**Comment by jcoglan** [comment]
@sdubinsky The `README` does mention that `rackup` should be run in production mode, but it might not be prominent enough. Are there other places in the docs where we could mention this to make it clearer?

**Comment by sdubinsky** [comment]
I don't think it needs to be any more obvious than it is, but the error message is so hideously useless that I think it would be good to have it noted in the readme like "if you don't, you'll get an invalid frame header error' or something, just to connect the problem and the solution.
